.travis.yml

```diff
@@ -15,17 +22,22 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-29.0.2
+    - build-tools-30.0.2
     - android-21
-    - android-29
+    - android-31
+    - android ${TARGET_VERSION}
     - extra-google-google_play_services
     - extra-google-m2repository
     - extra-android-m2repository
     - addon-google_apis-google-25
     - sys-img-armeabi-v7a-android-21
 
 before_install:
-  - yes | sdkmanager "platforms;android-28"
+  - touch $HOME/.android/repositories.cfg
+  - wget "https://dl.google.com/android/repository/commandlinetools-linux-7302050_latest.zip" -O commandlinetools.zip
```

Comment:
isn't there an easier way to install these tools?

Reply:
It didnt work at all after bumping the Java version
This fix was taken from Travis forums.